.github/workflows/actions.yml

```diff
@@ -0,0 +1,37 @@
+name: Actions
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'actions/**'
+  pull_request:
+    paths:
+    - 'actions/**'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  get-matrix:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Setup containers
+      uses: ./actions/setup
+      with:
+        path: sssd-ci-containers
+
+    - name: Setup containers
+      uses: ./actions/setup
+      with:
+        path: sssd-ci-containers
```

Comment:
The final result is clear, what I don't understand is why you add these lines in the first commit to remove them in the second.

Reply:
Thank you, this was just bad rebasing on my side.